sunpy/net/hek/tests/test_hek.py

```diff
@@ -223,10 +223,128 @@ def test_query_multiple_operators():
     assert len(results) == 7
 
 
+@pytest.mark.remote_data
+def test_astropy_unit_parsing(coronal_hole_search_result):
+    hek_attributes = _get_unit_attributes() + _get_coord_attributes()
+    attributes_with_unit = [attribute for attribute in hek_attributes if attribute.get("unit_prop", False)]
+    for attribute in attributes_with_unit:
+        if attribute["name"] in coronal_hole_search_result.colnames and not attribute.get("is_chaincode", False):
+            assert isinstance(coronal_hole_search_result[attribute['name']], u.Quantity)
+
+
+@pytest.mark.remote_data
+def test_chaincode_parsing(coronal_hole_search_result):
+    chaincode_attributes = [attribute for attribute in _get_coord_attributes() if attribute.get("is_chaincode", False)]
+    for attribute in chaincode_attributes:
+        if attribute["name"] in coronal_hole_search_result.colnames:
+            if isinstance(coronal_hole_search_result[attribute['name']], MaskedColumn):
+                assert all([isinstance(r, SkyCoord) for r in coronal_hole_search_result[attribute['name']]])
+            else:
+                assert isinstance(coronal_hole_search_result[attribute['name']], SkyCoord)
+
+
 @pytest.mark.remote_data
 def test_missing_times():
     # Check for https://github.com/sunpy/sunpy/pull/7627#issuecomment-2113451964
     client = hek.HEKClient()
     results = client.search(attrs.Time('2024-05-10', '2024-05-12'), attrs.hek.AR.NOAANum == 13664)
     assert isinstance(results["event_peaktime"][0], np.ma.core.MaskedConstant)
     assert results["event_peaktime"][3].isot == "2024-05-10T00:13:00.000"
+
+
+@pytest.fixture
```

Comment:
Adding this will mean we only make the query once.
```suggestion
@pytest.fixture(scope="session")
```
which should be fine as long as nothing modifies the result in place.